### ISSUE TITLE
Support VobSub

### ISSUE THREAD

**Issue by shamoke** [nixxquality/WebMConverter#60]
I made a rip of one of my dvds using a program called makemkv, and while it plays fine I encountered an error when trying to make a webm of it. I don't know if it's related but when copying the frame number from mpchc and pasting it into the trim function the time the video ended up on was way off. Full log below.

```
Arguments for pass 1: -f nut -i pipe:0   -an -c:v libvpx -threads 4 -slices 2 -metadata title="TENCHI_OVA_SERIES_1-7_t06" -quality best -lag-in-frames 16 -auto-alt-ref 1 -qmin 0 -crf 10 -qmax 30 -pass 1 -passlogfile "C:\Users\E\AppData\Local\Temp\ffmpeg2pass" -f webm -y "NUL"
Arguments for pass 2: -f nut -i pipe:0   -c:v libvpx -threads 4 -slices 2 -metadata title="TENCHI_OVA_SERIES_1-7_t06" -quality best -lag-in-frames 16 -auto-alt-ref 1 -qmin 0 -crf 10 -qmax 30 -pass 2 -passlogfile "C:\Users\E\AppData\Local\Temp\ffmpeg2pass" -f webm -y "C:\Users\E\Desktop\Tenchi OVA 1 ep7 - sample collection.webm"
--- CREATING AVISYNTH PROXY --- 
ffmpeg version N-63644-ge1bd40f Copyright (c) 2000-2014 the FFmpeg developers
  built on May 31 2014 22:08:58 with gcc 4.8.2 (GCC)
  configuration: --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-decklink --enable-zlib
  libavutil      52. 87.100 / 52. 87.100
  libavcodec     55. 65.100 / 55. 65.100
  libavformat    55. 42.100 / 55. 42.100
  libavdevice    55. 13.101 / 55. 13.101
  libavfilter     4.  5.100 /  4.  5.100
  libswscale      2.  6.100 /  2.  6.100
  libswresample   0. 19.100 /  0. 19.100
  libpostproc    52.  3.100 / 52.  3.100
[nut @ 0000000002aaf680] No main startcode found.
pipe:0: Invalid data found when processing input
--- FFMPEG HAS EXITED ---
--- ENTERING PASS 2 ---
--- CREATING AVISYNTH PROXY --- 
ffmpeg version N-63644-ge1bd40f Copyright (c) 2000-2014 the FFmpeg developers
  built on May 31 2014 22:08:58 with gcc 4.8.2 (GCC)
  configuration: --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-decklink --enable-zlib
  libavutil      52. 87.100 / 52. 87.100
  libavcodec     55. 65.100 / 55. 65.100
  libavformat    55. 42.100 / 55. 42.100
  libavdevice    55. 13.101 / 55. 13.101
  libavfilter     4.  5.100 /  4.  5.100
  libswscale      2.  6.100 /  2.  6.100
  libswresample   0. 19.100 /  0. 19.100
  libpostproc    52.  3.100 / 52.  3.100
[nut @ 0000000002aaf680] No main startcode found.
pipe:0: Invalid data found when processing input
--- FFMPEG HAS EXITED ---

ffmpeg.exe exited with exit code 1. That's usually bad.
If you have no idea what went wrong, open an issue on GitHub and copy paste the output of this window there.
```

**Comment by nixxquality** [comment]
Now that we're using mkvextract to extract attachments, we might as well use it to extract subtitles as well.
Expect full DVD subtitle support in the next version.

**Comment by nixxquality** [comment]
Included in [v2.17.0](https://github.com/nixxquality/WebMConverter/releases/tag/v2.17.0).
